identity/src/keypair.rs

```diff
@@ -27,7 +33,19 @@ use crate::error::{DecodingError, SigningError};
     feature = "rsa"
 ))]
 use crate::proto;
+#[cfg(any(
+    feature = "ecdsa",
+    feature = "secp256k1",
+    feature = "ed25519",
+    feature = "rsa"
+))]
 use quick_protobuf::{BytesReader, Writer};
+#[cfg(any(
+    feature = "ecdsa",
+    feature = "secp256k1",
+    feature = "ed25519",
+    feature = "rsa"
+))]
 use std::convert::TryFrom;
```

Comment:
Do we even need to import this? I thought that is part of the prelude starting with the 2018 or 2021 edition?